stdlib/_typeshed/__init__.pyi

```diff
@@ -189,3 +203,17 @@ else:
     @final
     class NoneType:
         def __bool__(self) -> Literal[False]: ...
+
+# This is an internal CPython type that is like, but subtly different from, a NamedTuple
+# Subclasses of this type are found in multiple modules.
+# See discussion at #6546
+class structseq(Sequence[_T_co], Generic[_T_co]):
```

Comment:
This inherits from tuple at runtime. It might be good to reflect that in stubs. Does this work?
```suggestion
class structseq(Tuple[_T_co, ...], Generic[_T_co]):
```

Reply:
Have made the change, thanks 👍